### ISSUE TITLE
Plugin load error

### ISSUE THREAD

**Issue by nomasprime** [neovim/nvim-lsp#231]
- `nvim --version`: 0.5.0
- nvim-lsp version(commit hash): b77a24f6b6bdf32d621f37c733f399f33a7eab0d
- `:checkhealth` result:
```
function health#check[21]..health#nvim_lsp#check, line 2
    Vim(lua):E5108: Error executing lua ...me/.config/nvim/plugged/nvim-lsp/lua/nvim_lsp/health.lua:3: loop or previous error loading module 'nvim_lsp/configs'
```
Every time I open NeoVim:
```
Error detected while processing /Users/nomasprime/.config/nvim/init.vim:
line  309:
E5108: Error executing lua ...rime/.config/nvim/plugged/nvim-lsp/lua/nvim_lsp/util.lua:13: attempt to call field 'empty_dict' (a nil value)
Error detected while processing /Users/nomasprime/.config/nvim/plugged/nvim-lsp/plugin/nvim_lsp.vim:
line   14:
E5108: Error executing lua [string ":lua"]:7: loop or previous error loading module 'nvim_lsp'
```
Config:
```
Plug 'neovim/nvim-lsp'

lua << END
  require'nvim_lsp'.solargraph.setup{}
END
```

**Comment by nomasprime** [comment]
Even installing the plugin with Plug I get:
```
Error detected while processing /Users/nomasprime/.config/nvim/plugged/nvim-lsp/plugin/nvim_lsp.vim:
line   14:
E5108: Error executing lua [string ":lua"]:7: loop or previous error loading module 'nvim_lsp'
```

**Comment by nomasprime** [comment]
FFS! Hours wasted because Homebrew wasn't actually updating NeoVim to the latest commit.

Anyway, working now.

**Comment by DShivansh** [comment]
I am getting this error on mac os I have nvim 0.6.1 can anyone please help?

**Comment by herko** [comment]
> 

Same here. MacOS + nvim 0.6.1.

**Comment by mjlbach** [comment]
You should reproduce with the minimal config.

**Comment by herko** [comment]
I fixed it, somehow. The thing was that at first I organized files to lua folders and probably something in this way of loading .lua files was wrong. I put everything into one single file (vim options + plugins loading) and suddenly no error was shown upon startup. So I started to divide these into separate files again and everything was ok.

I might load something before some dependency and that might cause the issue. Fixed on my end.
